src/middleware/issueDetails.middleware.js

```
@@ -284,16 +322,22 @@ export const getIssueDetails = renderTemplate({
   handlerName: 'getIssueDetails'
 })
 
+/* eslint-disable no-return-assign */
+const zeroEntityCount = (req) => req.entityCount = { entity_count: 0 }
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)